presets/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+if(ENABLE_PRESETS)
+
+    install(DIRECTORY
+            presets_bltc201
+            presets_eyetune
+            presets_milkdrop
+            presets_milkdrop_104
+            presets_milkdrop_200
+            presets_mischa_collection
+            presets_projectM
+            presets_stock
+            presets_tryptonaut
+            presets_yin
+            tests
```

Comment:
```suggestion
```

Reply:
Good one. They were installed with the autotools scripts, so wasn't sure if that was intentional. Be gone, then!

Reply:
Changed in commit cc785a6, now only installing these presets if ENABLE_TESTING is set to a true'ish value.